src/amoc_dist.erl

```diff
@@ -103,7 +103,6 @@ ceil(Number) ->
 ping_node(Node) ->
     case amoc_slave:ping(Node) of
```

Comment:
I think we no longer need the case now when we removed the `amoc_slave:monitor_master` on `pong` result.

Reply:
Yup, we clearly don't need it. I was thinking of removing it, thank for reminding :)